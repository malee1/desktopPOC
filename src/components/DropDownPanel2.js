import { React } from 'react';
import { makeStyles } from '@material-ui/core/styles';
import { Box, Typography, List, ListItem } from '@material-ui/core';
import { Info } from '@material-ui/icons';
// import { useSelector } from 'react-redux';
// import { panelOneLabels } from '../constants/panelLabels';

const useStyles = makeStyles({
  root: {
    flexGrow: 1,
    paddingTop: '5px'
  },
  box: {
    width: '27vw'
  },
  list: {
    flexDirection: 'column',
    whiteSpace: 'nowrap'
  },
  listItem: {
    backgroundColor: 'white'
  },
  iconContainer: {
    width: '8%',
    backgroundColor: 'white'
  },
  icon: {
    backgroundColor: 'white'
  },
  labelContainer: {
    width: '40%',
    marginLeft: '2px'
  },
  label: {
    backgroundColor: 'white',
    paddingLeft: '5px'
  },
  labelValueContainer: {
    width: '55%',
    marginLeft: '2px'
  },
  labelValue: {
    backgroundColor: 'white',
    paddingLeft: '5px',
    color: 'red'
  }
});

export default function PanelOne() {
  const classes = useStyles();

  return (
    <div className={classes.root}>
      <Box className={classes.box} bgcolor="primary.main">
        <List className={classes.list}>
          <ListItem className={classes.listItem}>
            <div className={classes.iconContainer}>
              <Info className={classes.icon} />
            </div>
            <div className={classes.labelContainer}>
              <Typography className={classes.label} variant="body1">
                More Info:
              </Typography>
            </div>
            <div className={classes.labelValueContainer}>
              <Typography className={classes.labelValue} variant="body1">
                Panel B
              </Typography>
            </div>
          </ListItem>
          <ListItem className={classes.listItem}>
            <div className={classes.iconContainer}>
              <Info className={classes.icon} />
            </div>
            <div className={classes.labelContainer}>
              <Typography className={classes.label} variant="body1">
                More Info:
              </Typography>
            </div>
            <div className={classes.labelValueContainer}>
              <Typography className={classes.labelValue} variant="body1">
                Panel B
              </Typography>
            </div>
          </ListItem>
          <ListItem className={classes.listItem}>
            <div className={classes.iconContainer}>
              <Info className={classes.icon} />
            </div>
            <div className={classes.labelContainer}>
              <Typography className={classes.label} variant="body1">
                More Info:
              </Typography>
            </div>
            <div className={classes.labelValueContainer}>
              <Typography className={classes.labelValue} variant="body1">
                Panel B
              </Typography>
            </div>
          </ListItem>
          <ListItem className={classes.listItem}>
            <div className={classes.iconContainer}>
              <Info className={classes.icon} />
            </div>
            <div className={classes.labelContainer}>
              <Typography className={classes.label} variant="body1">
                More Info:
              </Typography>
            </div>
            <div className={classes.labelValueContainer}>
              <Typography className={classes.labelValue} variant="body1">
                Panel B
              </Typography>
            </div>
          </ListItem>
        </List>
      </Box>
    </div>
  );
}
