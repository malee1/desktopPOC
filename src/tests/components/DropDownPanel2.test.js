import React from 'react';
import renderer from 'react-test-renderer';
import { render } from '@testing-library/react';

import DropDownPanel2 from '../../components/DropDownPanel2';

describe('the panel renders', () => {
  it('matches the snapshot', () => {
    const tree = renderer.create(<DropDownPanel2 />);
    expect(tree).toMatchSnapshot();
  });

  it('has one component', () => {
    const { getAllByTestId } = render(<DropDownPanel2 />);
    const container = getAllByTestId('drop-down-panel-2');
    expect(container.length).toBe(1);
  });
});
