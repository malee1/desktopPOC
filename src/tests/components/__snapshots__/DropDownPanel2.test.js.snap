// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the panel renders matches the snapshot 1`] = `
<div
  className="makeStyles-root-1"
  data-testid="drop-down-panel-2"
>
  <div
    className="MuiBox-root MuiBox-root-11 makeStyles-box-2"
  >
    <ul
      className="MuiList-root makeStyles-list-3 MuiList-padding"
    >
      <li
        className="MuiListItem-root makeStyles-listItem-4 MuiListItem-gutters"
        disabled={false}
      >
        <div
          className="makeStyles-iconContainer-5"
        >
          <svg
            aria-hidden={true}
            className="MuiSvgIcon-root makeStyles-icon-6"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm1 15h-2v-6h2v6zm0-8h-2V7h2v2z"
            />
          </svg>
        </div>
        <div
          className="makeStyles-labelContainer-7"
        >
          <p
            className="MuiTypography-root makeStyles-label-8 MuiTypography-body1"
          >
            More Info:
          </p>
        </div>
        <div
          className="makeStyles-labelValueContainer-9"
        >
          <p
            className="MuiTypography-root makeStyles-labelValue-10 MuiTypography-body1"
          >
            Panel B
          </p>
        </div>
      </li>
      <li
        className="MuiListItem-root makeStyles-listItem-4 MuiListItem-gutters"
        disabled={false}
      >
        <div
          className="makeStyles-iconContainer-5"
        >
          <svg
            aria-hidden={true}
            className="MuiSvgIcon-root makeStyles-icon-6"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm1 15h-2v-6h2v6zm0-8h-2V7h2v2z"
            />
          </svg>
        </div>
        <div
          className="makeStyles-labelContainer-7"
        >
          <p
            className="MuiTypography-root makeStyles-label-8 MuiTypography-body1"
          >
            More Info:
          </p>
        </div>
        <div
          className="makeStyles-labelValueContainer-9"
        >
          <p
            className="MuiTypography-root makeStyles-labelValue-10 MuiTypography-body1"
          >
            Panel B
          </p>
        </div>
      </li>
      <li
        className="MuiListItem-root makeStyles-listItem-4 MuiListItem-gutters"
        disabled={false}
      >
        <div
          className="makeStyles-iconContainer-5"
        >
          <svg
            aria-hidden={true}
            className="MuiSvgIcon-root makeStyles-icon-6"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm1 15h-2v-6h2v6zm0-8h-2V7h2v2z"
            />
          </svg>
        </div>
        <div
          className="makeStyles-labelContainer-7"
        >
          <p
            className="MuiTypography-root makeStyles-label-8 MuiTypography-body1"
          >
            More Info:
          </p>
        </div>
        <div
          className="makeStyles-labelValueContainer-9"
        >
          <p
            className="MuiTypography-root makeStyles-labelValue-10 MuiTypography-body1"
          >
            Panel B
          </p>
        </div>
      </li>
      <li
        className="MuiListItem-root makeStyles-listItem-4 MuiListItem-gutters"
        disabled={false}
      >
        <div
          className="makeStyles-iconContainer-5"
        >
          <svg
            aria-hidden={true}
            className="MuiSvgIcon-root makeStyles-icon-6"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm1 15h-2v-6h2v6zm0-8h-2V7h2v2z"
            />
          </svg>
        </div>
        <div
          className="makeStyles-labelContainer-7"
        >
          <p
            className="MuiTypography-root makeStyles-label-8 MuiTypography-body1"
          >
            More Info:
          </p>
        </div>
        <div
          className="makeStyles-labelValueContainer-9"
        >
          <p
            className="MuiTypography-root makeStyles-labelValue-10 MuiTypography-body1"
          >
            Panel B
          </p>
        </div>
      </li>
    </ul>
  </div>
</div>
`;
